test/compilationTests/milestonetracker/RLP.sol

```diff
@@ -376,7 +376,7 @@ library RLP {
  }
 
  // Assumes that enough memory has been allocated to store in target.
- function _copyToBytes(uint btsPtr, bytes memory tgt, uint btsLen) private view {
+ function _copyToBytes(uint btsPtr, bytes memory tgt, uint btsLen) private {
```

Comment:
Why is is non-view or even non-pure? It only does mstore/mload.

Reply:
As far as I remember it's the jump in the inline assembly that's not considered pure or even view at the moment (making sure that it always jumps to a piece of code that's view or pure is probably non-trivial).

Reply:
Right, that makes sense, we should document this properly and how to avoid it: use if/for/switch.